mix.exs

```diff
@@ -52,6 +52,7 @@ defmodule EpochElixir.Mixfile do
       {:credo, "~> 0.9.3", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 0.5", only: [:dev, :test], runtime: false},
       {:excoveralls, "~> 0.8.1", only: :test},
+      {:plug_cowboy, "~> 1.0"},
```

Comment:
why is this added?

Reply:
I couldn't start the project without plug_cowboy.

Reply:
Do you have an idea why this wasn't needed before?

Reply:
Maybe, there are some problems with Phoenix.